cypress-smoketests.json

```diff
@@ -5,6 +5,7 @@
     "toConsole": true
   },
   "video": true,
+  "videoUploadOnPasses": false,
```

Comment:
My understanding is this only disables the uploading of the video to Cypress' SASS offering.  The video still get's created though.  Right???

Reply:
Just stumbled on [this issue](https://github.com/cypress-io/cypress/issues/4345) that sadly confirms my understanding.  And here's the [documentation](https://docs.cypress.io/guides/references/configuration#Videos).

Reply:
@rgoshen and I late last week added the `videoCompression` key to this config file.  Maybe that value isn't small enough.  Maybe decrease it to `10`?

Reply:
sad. nevermind then.

Reply:
woops i didn't see your last comment. yeah, fair point about that being for the cypress dashboard which we aren't using. but by default it's set to true anyway and setting it to false still seems to reduce the time for the tests? running it one more time to see if the timer still remains lower.

updated `videoCompression` to 10

Reply:
it's also interesting to me that that issue is from [2019](https://github.com/cypress-io/cypress/issues/4345) and the comments on the open issue, suggesting this fix, are from [8 days ago](https://github.com/cypress-io/cypress/issues/17627#issuecomment-1116136937)

Reply:
I'm not sure 🤷  feels weird to do both probably. Wish we could get it to fail on an experimental

Reply:
Even though this is merged, thought this was pertinent! In the merged workflow, it looks like it's not compressing the videos anymore since the specs passed with the `videoUploadOnPasses` flag. [Before](https://app.circleci.com/pipelines/github/ustaxcourt/ef-cms/3399/workflows/8f828322-bc46-47b6-a19d-518da5bbb01a/jobs/35041) and [after](https://app.circleci.com/pipelines/github/ustaxcourt/ef-cms/3402/workflows/4c6b2b81-c69b-454f-8d29-ecbf593b0900/jobs/35132) workflows. Seems like this [paragraph](https://docs.cypress.io/guides/guides/screenshots-and-videos#:~:text=When%20using%20the,option%20to%20false.) in the docs helps support that 🧐